include/freerdp/channels/rdpecam.h

```diff
@@ -1,9 +1,8 @@
 /**
  * FreeRDP: A Remote Desktop Protocol Implementation
- * Video Optimized Remoting Virtual Channel Extension
+ * Camera Redirection Virtual Channel Extension
```

Comment:
Just a nit, feel free to ignore: While camera redirection is the accurate name of that extension, I would though go with `Video Capture Virtual Channel Extension`, as that is the name of the spec.